### PR TITLE
Import eight new revisions (the last year's worth) to upstream

### PR DESCRIPTION
**Pull request by 0cjs**
I simply ran your script to do this and it appears to have Just Worked.

The `master` branch in my repo is also rebased as per your instructions in the README and seems fine. I can do another PR for that if you like, if you don't want to do the rebase yourself; just let me know.

### CODE REVIEW

**Comment by 0cjs** [comment]
It turns out I didn't quite understand the user interface for the importer and missed a revision. I've corrected this and force-pushed both my `upstream` and `master` branches.

Just as a warning about the use of my `master` branch: I have a patch from the author for a fix to `.map` file output (as of bld151 it was not correctly replacing spaces with `\032` in the symbol values). I'm not sure if this is in current or if I'll need to apply it myself, but it will be going on my `master` branch soon and will remain there until bld153 (or another build with the fix) comes out.

**Comment by KubaO** [comment]
Thank you for your contribution!

**Comment by 0cjs** [comment]
Thanks for merging!

I don't know if you're on the `as-users` mailing list, but there's a new release coming out soon so I'll be putting in another PR for that unless you're going to do it.

**Comment by KubaO** [comment]
I wleczone PRs, and perhaps it’d be easiest if I added you to the project. I did this mirror out of sheer curiosity to learn from the source code; it’s not really fair that you have to ask someone to do real work as I didn’t really contribute anything yet.

Kuba

**Comment by 0cjs** [comment]
My main aim is to have a Git repo for AS that holds a vendor or upstream branch with regular imports of new releases from Alfred (with the latest release, for some reasonable definition of "release" always available at the head of `master`) and where I and others can have branches with patches on them. (You may have noticed that[my current `master`](https://github.com/0cjs/asl) has some patches I received from Alfred between the last and current releases.) Since you'd done most of the heavy lifting to get the upstream branch maintenance working (and thank you for that!), I just forked your repo and carried on with my work.

I think it's a great idea to do this in a single shared repo, instead of using forks, and if you're ok with this general plan and give me access I'm happy to continue my work in your repo and just delete my fork, which I think will help prevent confusion. Moving foward the plan I propose (which we of course can tweak as we go along) is:
1. Continue to add new releases as commits to `upstream`, and continue to rebase `master` on to that.
2. Create an `upstream-patches` branch, always rebased on top of the latest `upstream`, with any patches from Alfred that are likely to go in the next release.
3. Allow developers with access to this repo to create `dev/NAME/...` branches (e.g., `dev/0cjs/200108/playing-around` as necessary for experimentation, testing and whatnot.
4. Maybe look at a cleaner way of handling the `master` branch if we can think of one.

Basically, this repo would become the unofficial Git mirror for AS development and releases.

If you're ok with that, sure, give me access and I'll start doing this stuff along with you, and any others who are interested in joining.

Also, I strongly feel it's worth creating a [Gitter](https://gitter.im) channel for this repo for easier general conversation about it. I think that if you make me a developer on this repo I'll be able to do that and everybody with commit access to this repo would be a channel admin.

**Comment by 0cjs** [comment]
For what it's worth, I did create a gitter.im group for users of this assembler: https://gitter.im/Macroassembler-AS/. It's linked to this repo, so I think that everyone with commit access should also be an admin there.
